test/src/main/java/org/jvnet/hudson/test/JenkinsRule.java

```diff
@@ -911,6 +919,113 @@ public DumbSlave createSlave(Label l, EnvVars env) throws Exception {
     }
 
     /**
+     * Get JSON from A Jenkins endpoint.
+     * @param url The endpoint URL.
+     * @return The JSON.
+     */
+    public JSONWebResponse getJSON(@Nonnull String url) throws IOException, SAXException {
+        JenkinsRule.WebClient webClient = createWebClient();
+
+        if (url.startsWith("/")) {
+            url = url.substring(1);
```

Comment:
> it's not "wrong" to have it there (imo)

It is inconsistent with that is expected of `goTo` and similar methods, which specifically forbid an initial slash, so that callers must recognize that they are passing a context-relative path.